restructuredtext_lint/cli.py

```diff
@@ -12,7 +12,7 @@
     VERSION = version_file.read().strip()
 
 
-def _main(filepaths, format='text', stream=sys.stdout, encoding=None):
+def _main(filepaths, format='text', stream=sys.stdout, encoding=None, level=0, fail=0):
```

Comment:
Yes, I'm saying that `--level` should both be what we output as well as what is considered cause for failure. Imo, I made a poor design decision by not using "Silence is golden" for valid files =/